raiden/api/rest.py

```diff
@@ -594,6 +611,30 @@ def get_token_network_events(self, token_address, from_block, to_block):
         except InvalidBlockNumberInput as e:
             return api_error(str(e), status_code=HTTPStatus.CONFLICT)
 
+    def get_channel_history_events(
+            self,
+            token_address: typing.TokenAddress,
+            partner_address: typing.Address = None,
+    ):
+        try:
+            raiden_service_result = self.raiden_api.get_channel_history_events(
+                token_address,
+                partner_address,
+            )
+            result = []
+            for e in raiden_service_result:
+                name = e.get('event')
+                if name == 'EventTransferSentSuccess':
+                    event = self.event_transf_sent_succ_schema.dump(e)
+                elif name == 'EvnetTransferSentFailed':
+                    event = self.event_transf_sent_fail_schema.dump(e)
+                else:
+                    event = self.event_transf_rece_succ_schema.dump(e)
+                result.append(event.data)
+            return api_response(result=result)
+        except (InvalidBlockNumberInput, InvalidAddress) as e:
```

Comment:
most of the block inside try cannot raise these exceptions, reduce the try block only to the part of the code that can raise the exception